ext/storage_sources/gcp_store/test/test_gcp_connection.cpp

```diff
@@ -27,8 +27,261 @@
  */
 
 #define CATCH_CONFIG_MAIN
+#include "gcp_connection.h"
+
 #include <catch2/catch.hpp>
+#include <typeinfo>
```

Comment:
This was added to support a function that we added before, since we now deleted it I removed this include here:[8039651](https://github.com/wiredtiger/wiredtiger/pull/8749/commits/8039651071462d3bd21d784fdb335f39875e742f)